tests/conftest.py

```diff
@@ -1,12 +1,26 @@
+import os
 import pytest
 
 from pyrestcli.auth import NoAuthClient
 
 from carto.auth import APIKeyAuthClient
 from carto.users import UserManager
+from mocks import MockRequests, NotMockRequests
 
-from secret import USERNAME, API_KEY, ORGANIZATION
+if "API_KEY" in os.environ:
+    API_KEY = os.environ["API_KEY"]
+else:
+    from secret import API_KEY
```

Comment:
The use of python modules that are not in the repo is not a good pattern.

My advice would be to either load them from a json (it's cleaner), or have the tests fail if the environment is not set (the best option really).